packages/itwinui-react/src/core/SideNavigation/SideNavigation.tsx

```diff
@@ -57,6 +57,22 @@ type SideNavigationProps = {
    * @default false
    */
   isSubmenuOpen?: boolean;
+  /**
+   * Passes props for SideNav.
+   */
+  sideNavProps?: React.ComponentProps<'div'>;
```

Comment:
this could be named `innerProps`? it is an inner-wrapper inside the outer-wrapper